my-app/src/App.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+
+
+class GameBoard {
+  constructor( size ){
+    this.size = size
+    this.board = []
+    this.initializeBoard()
+  }
+
+  initializeBoard(){
+    for(let x = 0; x < this.size; x++){
+      this.board[x] = []
+      for (let y = 0; y < this.size; y++){
+        this.board[x][y] = new Gamepeice(x,y)
+      }
+    }
+    return this.board
+  }
+}
+
+class Gamepeice {
+  constructor(x,y) {
+    this.x = x
+    this.y = y
+    this.value = 'blank'
+  }
+}
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.state = {
+      wIsNext: true,
```

Comment:
Your code is very readable. One minor improvement would be to practice avoiding abbreviating, shifting `wIsNext` to `whiteIsNext`.